antora/docs/modules/ROOT/pages/index.adoc

```diff
@@ -15,7 +15,7 @@ xref:pipeline_policy.adoc[Pipeline Policy].
 
 === Additional Documentation
 
-* https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/[HACBS Documentation]
+* https://redhat-appstudio.github.io/docs.stonesoup.io/[HACBS Documentation]
```

Comment:
```suggestion
* https://redhat-appstudio.github.io/docs.stonesoup.io/[Stonesoup Documentation]
```

Reply:
Addressing in https://github.com/hacbs-contract/ec-policies/pull/363